internal/cli/email_templates.go

```diff
@@ -0,0 +1,292 @@
+package cli
+
+import (
+	"fmt"
+
+	"github.com/auth0/auth0-cli/internal/ansi"
+	"github.com/spf13/cobra"
+	"gopkg.in/auth0.v5"
+	"gopkg.in/auth0.v5/management"
+)
+
+const (
+	emailTemplateVerifyLink     = "verify-link"
+	emailTemplateVerifyCode     = "verify-code"
+	emailTemplateChangePassword = "change-password"
+	emailTemplateWelcome        = "welcome"
+	emailTemplateBlockedAccount = "blocked-account"
+	emailTemplatePasswordBreach = "password-breach"
+	emailTemplateMFAEnrollment  = "mfa-enrollment"
+	emailTemplateMFACode        = "mfa-code"
+	emailTemplateUserInvitation = "user-invitation"
+)
+
+var (
+	emailTemplateTemplate = Argument{
+		Name: "Template",
+		Help: fmt.Sprintf("Template name. Can be '%s', '%s', '%s', '%s', '%s', '%s', '%s', '%s' or '%s'",
+			emailTemplateVerifyLink,
+			emailTemplateVerifyCode,
+			emailTemplateChangePassword,
+			emailTemplateWelcome,
+			emailTemplateBlockedAccount,
+			emailTemplatePasswordBreach,
+			emailTemplateMFAEnrollment,
+			emailTemplateMFACode,
+			emailTemplateUserInvitation),
+	}
+
+	emailTemplateBody = Flag{
+		Name:       "Body",
+		LongForm:   "body",
+		ShortForm:  "b",
+		Help:       "Body of the email template.",
+		IsRequired: true,
+	}
+
+	emailTemplateFrom = Flag{
+		Name:         "From",
+		LongForm:     "from",
+		ShortForm:    "f",
+		Help:         "Sender's 'from' email address.",
+		AlwaysPrompt: true,
+	}
+
+	emailTemplateSubject = Flag{
+		Name:         "Subject",
+		LongForm:     "subject",
+		ShortForm:    "s",
+		Help:         "Subject line of the email.",
+		AlwaysPrompt: true,
+	}
+
+	emailTemplateEnabled = Flag{
+		Name:         "Enabled",
+		LongForm:     "enabled",
+		ShortForm:    "e",
+		Help:         "Whether the template is enabled (true) or disabled (false).",
+		AlwaysPrompt: true,
+	}
+
+	emailTemplateURL = Flag{
+		Name:      "Result URL",
+		LongForm:  "url",
+		ShortForm: "u",
+		Help:      "URL to redirect the user to after a successful action.",
+	}
+
+	emailTemplateLifetime = Flag{
+		Name:      "Result URL Lifetime",
+		LongForm:  "lifetime",
+		ShortForm: "l",
+		Help:      "Lifetime in seconds that the link within the email will be valid for.",
+	}
+
+	emailTemplateOptions = pickerOptions{
+		{"Verification Email (using Link)", emailTemplateVerifyLink},
+		{"Verification Email (using Code)", emailTemplateVerifyCode},
+		{"Change Password", emailTemplateChangePassword},
+		{"Welcome Email", emailTemplateWelcome, },
+		{"Blocked Account Email", emailTemplateBlockedAccount},
+		{"Password Breach Alert", emailTemplatePasswordBreach},
+		{"Enroll in Multifactor Authentication", emailTemplateMFAEnrollment},
+		{"Verification Code for Email MFA", emailTemplateMFACode},
+		{"User Invitation", emailTemplateUserInvitation},
+	}
```

Comment:
These are the same strings used in Manage:

<img width="596" alt="Screen Shot 2021-06-30 at 00 08 23" src="https://user-images.githubusercontent.com/5055789/124052457-d4c72280-d9f4-11eb-92da-56b2a264828e.png">